CONTRIBUTING.md

```diff
@@ -53,9 +53,9 @@ Get-Help ./build.ps1 -Detailed
 
 Pester has a C# Solution which requires .NET Framework SDKs and Developer Packs in order to compile. The targeted frameworks can be found in `src\csharp\Pester\Pester.csproj`.
 
-### Install .NET Core 6.0 SDK
+### Install .NET 8.0 SDK
 
-[Download Link](https://dotnet.microsoft.com/download/dotnet-core/6.0)
+[Download Link](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
 
 ### .Net Framework 4.5 Developer Pack
```

Comment:
Can this be removed? Necessary now that we use `Microsoft.NETFramework.ReferenceAssemblies` package in assembly?

Reply:
Probably not (I have all of them installed so I cannot easily check), but the whole point of that nuget is that you can build .NET Framework even on Linux where it cannot run (without mono).

Reply:
Cannot make suggestion that deletes it, please delete it.